#! /bin/sh -e

# DP: add rcapid manpage

dir=
if [ $# -eq 3 -a "$2" = '-d' ]; then
    pdir="-d $3"
    dir="$3/"
elif [ $# -ne 1 ]; then
    echo >&2 "usage: `basename $0`: -patch|-unpatch [-d <srcdir>]"
    exit 1
fi
case "$1" in
    -patch)
        patch $pdir -f --no-backup-if-mismatch -p0 < $0
        ;;
    -unpatch)
        patch $pdir -f --no-backup-if-mismatch -R -p0 < $0
	rm -f rcapid/rcapid.8
        ;;
    *)
	echo >&2 "usage: `basename $0`: -patch|-unpatch [-d <srcdir>]"
        exit 1
esac
exit 0

diff -urN rcapid.old/Makefile.am rcapid/Makefile.am
--- rcapid.old/Makefile.am	2002-07-19 23:23:54.000000000 +0200
+++ rcapid/Makefile.am	2004-07-11 15:08:36.000000000 +0200
@@ -13,5 +13,6 @@
 LDADD = -lcapi20
 
 sbin_PROGRAMS = rcapid
+man_MANS = rcapid.8
 
 rcapid_SOURCES = rcapid.c
diff -urN rcapid.old/rcapid.8 rcapid/rcapid.8
--- rcapid.old/rcapid.8	1970-01-01 01:00:00.000000000 +0100
+++ rcapid/rcapid.8	2002-07-04 13:45:55.000000000 +0200
@@ -0,0 +1,36 @@
+.\" $Id: rcapid.8,v 1.1 2002/07/04 11:45:55 paul Exp $
+.\"
+.\" CHECKIN $Date: 2002/07/04 11:45:55 $
+.\"
+.TH rcapid 8
+.SH NAME
+rcapid \- remote CAPI server
+.SH SYNOPSIS
+rcapid [-l loglevel]
+.SH DESCRIPTION
+.B rcapid
+is a server to communicate with CAPI clients.
+
+It is designed to run from inetd.
+
+add the following line to /etc/services:
+
+  capi 6000/tcp
+
+add the following line to /etc/inetd.conf:
+
+  capi stream tcp nowait.9999 /usr/sbin/rcapid rcapid
+
+(there's not much documentation to be found. help!)
+
+.SH OPTIONS
+.TP
+.BI "-l" " loglevel"
+the level of verbosity in the logging.
+This is logged in /tmp/rcapid.log
+
+.SH BUGS
+The logging should go through syslog.
+
+.SH AUTHOR
+manpage written from C source and README by Paul Slootman <paul@debian.org>.
